hypothesis-python/src/hypothesis/strategies/_internal/core.py

```diff
@@ -220,6 +236,7 @@ def accept(*args, **kwargs):
 
 defines_strategy = base_defines_strategy(False)
 defines_strategy_with_reusable_values = base_defines_strategy(True)
+defines_strategy_without_laziness = base_defines_strategy(False, try_non_lazy=True)
```

Comment:
Also no need to do this in this PR but I kinda feel like we should just inline all of these.